### DISCUSSION TITLE
Empty lines mess up code snippet

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#915] [Q&A] [answered]
This is what metrics generates:
![image](https://user-images.githubusercontent.com/42429413/155865791-21cac514-97ab-4b2b-892b-066cc1343a69.png)
while this is what the actual diff is: https://github.com/Nixinova/Linguist/commit/22e379f039b5ffc79b81be45639230537726126e
![image](https://user-images.githubusercontent.com/42429413/155865795-d093ce16-6e21-44d0-ba9d-0b14ea954540.png)

There's an erroneous space before `+## Next` and the line `## 2.4.1` should not be highlighted: they've merged into the preceeding empty lines

**Comment by lowlighter** [comment, marked as answer]
Should have been fixed by 60857e1480e2217b9387508015be1b966c3e7da2
